shaders/position.yaml

```diff
@@ -128,10 +129,11 @@ layers:
         draw:
             polygons:
                 style: buildings
-                order: 4
+                order: function() { return feature.sort_key; }
                 color: [.7,.7,.7]
                 extrude: true
             outline:
+                order: function() { return feature.sort_key + 1; }
```

Comment:
This is probably backwards -- if you draw an outline over the intended inline, you won't see the inline.